php/Terminus/Models/User.php

```diff
@@ -32,7 +32,8 @@ public function __construct($attributes, $options = array()) {
     $this->workflows   = new Workflows(
       array('owner' => $this)
     );
-    $this->instruments = new Instruments(array('user' => $this));
+    $this->instruments   = new Instruments(array('user' => $this));
+    $this->organizations = new UserOrganizationMemberships(array('user' => $this));
```

Comment:
I think we should rename this attribute to be `organizationMemberships` to be clear that it's a collection of memberships, not organization objects.